wkcuber/README.md

```diff
@@ -1,155 +1,49 @@
-* `wkcuber.cubing`: Convert image stacks (e.g., `tiff`, `jpg`, `png`, `bmp`, `dm3`, `dm4`) to WKW cubes
-* `wkcuber.tile_cubing`: Convert tiled image stacks (e.g. in `z/y/x.ext` folder structure) to WKW cubes
-* `wkcuber.convert_knossos`: Convert KNOSSOS cubes to WKW cubes
-* `wkcuber.convert_nifti`: Convert NIFTI files to WKW files (Currently without applying transformations).
-* `wkcuber.convert_raw`: Convert RAW binary data (.raw, .vol) files to WKW datasets
-* `wkcuber.downsampling`: Create downsampled magnifications (with `median`, `mode` and linear interpolation modes). Downsampling compresses the new magnifications by default (disable via `--no_compress`).
-* `wkcuber.compress`: Compress WKW cubes for efficient file storage (especially useful for segmentation data)
-* `wkcuber.metadata`: Create (or refresh) metadata (with guessing of most parameters)
-* `wkcuber.recubing`: Read existing WKW cubes in and write them again specifying the WKW file length. Useful when dataset was written e.g. with file length 1.
-* `wkcuber.check_equality`: Compare two WKW datasets to check whether they are equal (e.g., after compressing a dataset, this task can be useful to double-check that the compressed dataset contains the same data).
-* Most modules support multiprocessing
+wkcuber offers some commands to work with WEBKNOSSOS datasets:
+
+- `wkcuber compress`: Compress a WEBKNOSSOS dataset
+- `wkcuber convert`: Convert a image stack to a WEBKNOSSOS dataset
+- `wkcuber download`: Download a WEBKNOSSOS dataset from a remote location
+- `wkcuber downsample`: Downsample a WEBKNOSSOS dataset
+- `wkcuber upload`: Upload a local WEBKNOSSOS dataset to a remote location
+- `wkcuber upsample`: Upsample a WEBKNOSSOS dataset
 
 ## Supported input formats
 
-* Standard image formats, e.g. `tiff`, `jpg`, `png`, `bmp`
-* Proprietary image formats, e.g. `dm3`
-* Tiled image stacks (used for Catmaid)
-* KNOSSOS cubes
-* NIFTI files
-* Raw binary files
+- Standard image formats, e.g. `tiff`, `jpg`, `png`, `bmp`
+- Proprietary image formats, e.g. `dm3`
+- Raw binary files
 
 ## Installation
-### Python 3 with pip from PyPi
-- `wkcuber` requires at least Python 3.8
-
-```bash
-# Make sure to have lz4 installed:
-# Mac: brew install lz4
-# Ubuntu/Debian: apt-get install liblz4-1
-# CentOS/RHEL: yum install lz4
-
-pip install wkcuber
-```
-
-### Docker
-Use the CI-built image: [scalableminds/webknossos-cuber](https://hub.docker.com/r/scalableminds/webknossos-cuber/). Example usage `docker run -v <host path>:/data --rm scalableminds/webknossos-cuber wkcuber --layer_name color --scale 11.24,11.24,25 --name great_dataset /data/source/color /data/target`.
 
+### Python 3 with pip from PyPi
 
-## Usage
+- `wkcuber` requires at least Python 3.10
 
 ```bash
-# Convert arbitrary, supported input files into wkw datasets. This sets reasonable defaults, but see other commands for customization.
-python -m wkcuber \
```

Comment:
Some instructions/examples for how to use the wkcuber are necessary in the readme.

Reply:
I added some examples in the docs.